vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -3192,6 +3194,32 @@ func TestAccResourceVSphereVirtualMachine_interpolatedDisk(t *testing.T) {
 	})
 }
 
+func TestAccResourceVSphereVirtualMachine_deployOvfFromUrl(t *testing.T) {
+
+	vmName := "terraform_test_vm_" + acctest.RandStringFromCharSet(4, acctest.CharSetAlphaNum)
+	remoteOvfUrl := os.Getenv("REMOTE_OVF_URL")
```

Comment:
I see it is already an env variable. We should remove the default and make the env variable mandatory.